package.json

```diff
@@ -117,7 +117,7 @@
     "ts-jest": "^26.0.0",
     "ts-node": "^9.0.0",
     "tsconfig-paths": "^3.10.1",
-    "typescript": "^5.0.4",
+    "typescript": "~5.3.3",
```

Comment:
Could you explain this change?

Reply:
I'd like to consider this as a separate PR. It's outside the scope of what you're trying to do here and I think for our case, that ESLint warning is benign.

Reply:
Done! Thank you very much for pointing out! :smile: